### PR TITLE
resource actions everywhere

### PR DESCRIPTION
**Pull request by der-john**
This replaces the last few usages of `moving-column-menu-button` with resourceActions. It's necessary for implementing resource headers.

Unfortunately, none of this is tested due to backend bug #2513.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div data-ng-switch="transclusionId">
     <div data-ng-switch-when="menu">
-        <i class="icon-document moving-column-icon"></i>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html]: Why did you remove this?

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-        <a class="moving-column-menu-button" data-ng-if="proposalItemOptions.POST" data-ng-href="{{ documentUrl | adhParentPath | adhResourceUrl:'edit' }}">{{ "TR__EDIT" | translate }}</a>
-
+        <adh-resource-actions
+            data-resource-path="{{documentUrl | adhParentPath}}"
+            data-edit="true"></adh-resource-actions>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html]: Resource actions should be in moving column body, not header

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/ProposalEditColumn.html [hunk 3]
```diff
@@ -3,8 +3,10 @@
         <div class="moving-column-tab">
             <i class="icon-document moving-column-icon"></i>
         </div>
-        <a class="moving-column-menu-button" data-ng-href="{{ proposalUrl | adhParentPath | adhResourceUrl }}">{{ "TR__CANCEL" | translate }}</a>
-
+        <adh-resource-actions
+            data-resource-path="{{proposalUrl | adhParentPath}}"
+            data-cancel="true">
+        </adh-resource-actions>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/ProposalEditColumn.html]: should be in body, not header

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/ProposalImageColumn.html [hunk 4]
```diff
@@ -3,8 +3,10 @@
         <div class="moving-column-tab">
             <i class="icon-document moving-column-icon"></i>
         </div>
-        <a class="moving-column-menu-button" data-ng-href="{{ proposalUrl | adhParentPath | adhResourceUrl }}">{{ "TR__CANCEL" | translate }}</a>
-
+        <adh-resource-actions
+            data-resource-path="{{documentUrl | adhParentPath}}"
+            data-cancel="true">
+        </adh-resource-actions>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/ProposalImageColumn.html]: should be in body, not header

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Alexanderplatz/Workbench/ProcessDetailColumn.html [hunk 5]
```diff
@@ -6,12 +6,11 @@
         <div class="moving-column-tab" data-ng-if="data.isShowMap">
             <i class="icon-map-filled moving-column-icon"></i>
         </div>
-
-        <a
-            class="moving-column-menu-button"
+        <adh-resource-actions
+            data-ng-if="processOptions.canPost(documentItemType) && tab === 'documents'"
             data-ng-click="setCameFrom()"
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Alexanderplatz/Workbench/ProcessDetailColumn.html]: there should not be a click handler on the whole directive

**Comment by xi** [comment]
The `.proposal-form` overlaps with the resource actions:

2016-07-13-150314_1440x900_scrot

I will fix this separately.

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Alexanderplatz/Workbench/ProcessDetailColumn.html (hunk 5 above)
Comment (der-john) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Alexanderplatz/Workbench/ProcessDetailColumn.html]: it does. thanks.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html (hunk 1 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/DebateWorkbench/DocumentDetailColumn.html]: by accident.
